2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Configure stats pid to 178097
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/.config/wandb/settings
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/settings
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_init.py:_log_setup():532] Logging user logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_142054-itjl729z/logs/debug.log
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_init.py:_log_setup():533] Logging internal logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_142054-itjl729z/logs/debug-internal.log
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_init.py:init():617] calling init triggers
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_init.py:init():667] starting backend
2024-10-15 14:20:54,340 INFO    MainThread:178097 [wandb_init.py:init():671] sending inform_init request
2024-10-15 14:20:54,341 INFO    MainThread:178097 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-15 14:20:54,341 INFO    MainThread:178097 [wandb_init.py:init():684] backend started and connected
2024-10-15 14:20:54,343 INFO    MainThread:178097 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-15 14:20:54,343 INFO    MainThread:178097 [wandb_init.py:init():779] updated telemetry
2024-10-15 14:20:54,353 INFO    MainThread:178097 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-15 14:21:04,683 WARNING MainThread:178097 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 815, in init
    result = run_init_handle.wait(
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-10-15 14:21:09,507 INFO    MainThread:178097 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Configure stats pid to 178097
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/.config/wandb/settings
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/settings
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_init.py:_log_setup():532] Logging user logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_142109-7aira1p3/logs/debug.log
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_init.py:_log_setup():533] Logging internal logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_142109-7aira1p3/logs/debug-internal.log
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_init.py:init():617] calling init triggers
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_init.py:init():667] starting backend
2024-10-15 14:21:09,510 INFO    MainThread:178097 [wandb_init.py:init():671] sending inform_init request
2024-10-15 14:21:09,511 INFO    MainThread:178097 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-15 14:21:09,511 INFO    MainThread:178097 [wandb_init.py:init():684] backend started and connected
2024-10-15 14:21:09,513 INFO    MainThread:178097 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-15 14:21:09,513 INFO    MainThread:178097 [wandb_init.py:init():779] updated telemetry
2024-10-15 14:21:09,522 INFO    MainThread:178097 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-15 14:42:49,275 WARNING MainThread:178097 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 815, in init
    result = run_init_handle.wait(
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-10-15 14:43:07,759 INFO    MainThread:178097 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Configure stats pid to 178097
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/.config/wandb/settings
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/settings
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_init.py:_log_setup():532] Logging user logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_144307-o80u6nfl/logs/debug.log
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_init.py:_log_setup():533] Logging internal logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_144307-o80u6nfl/logs/debug-internal.log
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_init.py:init():617] calling init triggers
2024-10-15 14:43:07,762 INFO    MainThread:178097 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-15 14:43:07,763 INFO    MainThread:178097 [wandb_init.py:init():667] starting backend
2024-10-15 14:43:07,763 INFO    MainThread:178097 [wandb_init.py:init():671] sending inform_init request
2024-10-15 14:43:07,763 INFO    MainThread:178097 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-15 14:43:07,764 INFO    MainThread:178097 [wandb_init.py:init():684] backend started and connected
2024-10-15 14:43:07,766 INFO    MainThread:178097 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-15 14:43:07,766 INFO    MainThread:178097 [wandb_init.py:init():779] updated telemetry
2024-10-15 14:43:07,775 INFO    MainThread:178097 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-15 14:43:08,071 WARNING MsgRouterThr:178097 [router.py:message_loop():77] message_loop has been closed
2024-10-15 14:43:08,072 WARNING MsgRouterThr:178097 [router.py:message_loop():77] message_loop has been closed
2024-10-15 14:43:08,072 WARNING MsgRouterThr:178097 [router.py:message_loop():77] message_loop has been closed
2024-10-15 14:44:20,557 WARNING MainThread:178097 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 815, in init
    result = run_init_handle.wait(
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-10-15 16:06:15,697 INFO    MainThread:178097 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-15 16:06:15,702 INFO    MainThread:178097 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-15 16:06:15,705 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-15 16:06:15,705 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Configure stats pid to 178097
2024-10-15 16:06:15,705 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/.config/wandb/settings
2024-10-15 16:06:15,705 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/settings
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_init.py:_log_setup():532] Logging user logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_160615-bl9ofkjy/logs/debug.log
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_init.py:_log_setup():533] Logging internal logs to /home/hgj/SDM274/HA-03&04_Classification_and_Logistic_Regression/wandb/run-20241015_160615-bl9ofkjy/logs/debug-internal.log
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_init.py:init():617] calling init triggers
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_init.py:init():667] starting backend
2024-10-15 16:06:15,706 INFO    MainThread:178097 [wandb_init.py:init():671] sending inform_init request
2024-10-15 16:06:15,707 INFO    MainThread:178097 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-15 16:06:15,707 INFO    MainThread:178097 [wandb_init.py:init():684] backend started and connected
2024-10-15 16:06:15,709 INFO    MainThread:178097 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-15 16:06:15,709 INFO    MainThread:178097 [wandb_init.py:init():779] updated telemetry
2024-10-15 16:06:15,720 INFO    MainThread:178097 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-15 16:06:29,868 WARNING MainThread:178097 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 815, in init
    result = run_init_handle.wait(
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/hgj/anaconda3/envs/sdm274/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
